go-app.js

```diff
@@ -22,6 +22,12 @@ go.app = function() {
         self.init = function() {            
             self.http = new JsonApi(self.im);       
         };
+
+        function capitaliseLocation(string) {
+            return string.replace(/\w\S*/g, function(s){
+                return s.charAt(0).toUpperCase() + s.substr(1).toLowerCase();
+            });
+        }
```

Comment:
We probably want a unit tests for this function to make sure that it's doing what we want it to do.